SKNIBot/Core/Database/Models/DynamicDB/DateMessageStats.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SKNIBot.Core.Database.Models.DynamicDB
+{
+    public class DateMessageStat
+    {
+        public DateMessageStat()
+        {
+            count = 0;
+            ChannelID = 0;
+            Date = "Unknown";
+        }
+        public DateMessageStat(int cnt, ulong channel, string date)
+         {
+             count = cnt;
+             ChannelID = channel;
+             Date = date;
+         }
+        public int ID { get; set; }
+        public int ServerID { get; set; }
+        public ulong ChannelID { get; set; }
```

Comment:
In SQLite Integer can use 8 bytes, but it's signed integer. Are you sure putting there ulong won't overflow it causing issues?
https://www.sqlite.org/datatype3.html
Previously we used string to keep ulong Discord ids.